tests/plots/test_declarative.py

```diff
@@ -1553,6 +1553,31 @@ def test_declarative_contour_label_fontsize():
     return pc.figure
 
 
+@pytest.mark.mpl_image_compare(remove_text=True, tolerance=0)
+@needs_cartopy
+def test_declarative_raster():
+    """Test making a raster plot."""
+    data = xr.open_dataset(get_test_data('narr_example.nc', False))
+
+    raster = RasterPlot()
+    raster.data = data
+    raster.field = 'Temperature'
+    raster.level = 700 * units.hPa
+
+    panel = MapPanel()
+    panel.area = 'us'
+    panel.projection = 'lcc'
+    panel.layers = ['coastline', 'borders', 'usstates']
```

Comment:
We could probably reduce the number of layers to avoid future issues with changes to boundaries. Let's go with just 'coastline'.